__tests__/integration.test.js

```diff
@@ -175,4 +175,72 @@ describe("GET requests", () => {
         });
     });
   });
+
+  describe("/api/articles/:article_id/comments", () => {
+    it('returns with status code 200 "OK"', () => {
+      return request(app).get("/api/articles/1/comments").expect(200);
+    });
+
+    it("responds with an array of comment objects", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(Array.isArray(comments)).toBe(true);
+          expect(comments.length).toBeGreaterThan(0);
+
+          for (let i = 0; i < comments.length; i++) {
+            const isObject = Object.keys(comments[i]).length > 0;
+            const isNotArray = !Array.isArray(comments[i]);
+
+            expect(isObject && isNotArray).toBe(true);
+          }
+        });
+    });
+
+    it("returned objects contain the required properties", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .then(({ body: { comments } }) => {
+          return comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                article_id: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
+
+    it("returned array should be sorted by most recent comments", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .then(({ body: { comments } }) => {
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+
+    it("returns a 404 if no comments are found OR if the ID does not exist", () => {
```

Comment:
I think these would be better suited in 2 different tests as the 404 error is being thrown for different reasons